doc/contributing/maintaining-types-for-nodejs.md

```diff
@@ -28,6 +28,54 @@ The agreement was that the ideal flow would be as follows:
 * Automation within external type projects consumes the JSON and automatically
   generates a PR to add the API.
 
+## High level approach - development workflow
+
+The number of people using TypeScript with Node.js is significant enough
+that providing a good developer experience is important. While TypeScript
+is identified specifically, a secondary goal is that what we provide to improve
+development experience with TypeScrtipt would apply to other type
+systems as well.
+
+We have agreed that the approach will `NOT` include bundling TypeScript
```

Comment:
```suggestion
We have agreed that the approach will *not* include bundling TypeScript
```

Reply:
Duplicate of https://github.com/nodejs/node/pull/44030#discussion_r932613759.